boards/x86/galileo/galileo.dts

```diff
@@ -42,3 +42,9 @@
 &spi0 {
 	status = "ok";
 };
+
+#ifdef CONFIG_SHARED_IRQ_0
```

Comment:
ifdef's in dts aren't allowed, why do we need this ifdef?

Reply:
Will remove in next rebase.

Reply:
Removed the #ifdef in the newest code.